src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -617,6 +618,8 @@ def configure_worker_command(
         """Get the command to configure the Worker. This must be run as root."""
 
         cmds = [
+            "source /opt/deadline/worker/bin/activate",
+            "AWS_DEFAULT_REGION={self.configuration.region}",
```

Comment:
Is this supposed to be an f-string?

Reply:
Yup! Will fix. Good eye. It must have defaulted, correctly, to `us-west-2` when the env var contained nonsense.